SETUP/INSTALL.md

```diff
@@ -57,10 +57,10 @@ These middleware components match the following major distribution releases:
 
 ## Browser support
 The following are the lowest known supported browser versions for the code:
-* Chrome 54
+* Chrome 57
```

Comment:
Since we're here, the codename for Ubuntu 20.04 above is incorrect, it should be Focal, not Bionic.